components/button.css

```diff
@@ -0,0 +1,96 @@
+.button_main {
+  border: 1px solid var(--gray-border);
+  border-radius: 5px;
+  margin: var(--margin-width);
+  padding: 1rem;
+  display: flex;
+}
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 1rem 1.5rem;
+}
+
+.btn-primary {
+  border: 1px solid var(--gray-border);
+  padding: 0.5rem 1rem;
+  color: var(--white);
+  line-height: 1.75;
+  font-weight: 600;
+  letter-spacing: 0.02857em;
+  border-radius: 7px;
+  cursor: pointer;
+}
+.btn-text {
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  line-height: 1.75;
+  font-weight: 600;
+  letter-spacing: 0.02857em;
+}
+.btn-secondary,
+.btn-icon {
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  line-height: 1.75;
+  font-weight: 600;
+  letter-spacing: 0.02857em;
+  border: 1px solid;
+  border-radius: 7px;
+  padding: 0.45rem 1rem;
+}
+.btn-bg-color {
+  background-color: var(--primary-color);
+}
+.action-button {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 1rem 1.5rem;
```

Comment:
The `.action-button` class and the `.button` both have the same code, you should avoid duplicate classes